### ISSUE TITLE
Idea: Provide a command line tool

### ISSUE THREAD

**Issue by hroncok** [bkabrda/anymarkup#9]
First of all, i don't need this, I'm just proposing this on the "nice to have" level.
I think it could help anymarkup to gain more glory.
## The ultimate `/usr/bin/anymarkup`

Reads from stdin or file, outputs to stdout or file. Has `--from` and `--to` optional format switches, guesses the format otherwise based on file extension and/or provided content. (Actually when writing to stdout, `--to` needs to be mandatory).

All the logic is already there, all this needs is a nice entry point and click.

Imagine the glory of this tool, you could do:

```
curl http://fedora.portingdb.xyz/stats.json | anymarkup --to yaml | anymarkup --to json
```

I might provide pull request if you approve the idea.

**Comment by bkabrda** [comment]
I like this idea. I have actually been thinking about something like this for quite some time, but I never got the time to do it. Some notes:

* should probably read from stdin and write to stdout by default, unless input file is provided; output file should be provided using e.g. `-o file.yaml`
* if possible, please avoid click; I know it's nice, but I'd like to avoid the dependency - argparse should be ok for simple stuff like this

If you (or anyone else) want to work on this, feel free to do so, but please ping this issue first, so that anyone else doesn't pick it up in the meanwhile.

**Comment by morph027** [comment]
Found jq wrapper https://github.com/kislyuk/yq which at leasts supports json, xml and yaml and allows to just pipe stdout/stdin into each other like:

`xq . foobar.xml | yq -y .`

**Comment by nightwatchcyber** [comment]
I am adding a pull request with the CLI and tests

**Comment by bkabrda** [comment]
Implemented in #10
